catalog/catalog.bom

```diff
@@ -43,7 +43,7 @@ brooklyn.catalog:
 
       brooklyn.config:
         install.version: 3.2
-        runtimeTemplates:
+        runtimeFiles:
```

Comment:
My preference is to keep the file as a template. Is there a good reason to change? I appreciate that the `${"$"}{__P(..)}` syntax is a bit ungainly.

As an alternative if you feel strongly, perhaps we could change the contract of the entity from requiring the remote resource (and hard-coding `plan.jmx`) to requiring the path of the plan on the instance and a promise that it will end up there. In this case the configuration on `jmeter-app` would change from:
```
brooklyn.config:
  plan: classpath://resources/load.jmx
```
to
```
brooklyn.config:
  plan: load.jmx
  runtimeTemplates:
    classpath://resources/load.jmx: load.jmx    
```
And in your non-templated case you would write:
```
brooklyn.config:
  plan: foo.jmx
  runtimeFiles:
    classpath://resources/foo.jmx: foo.jmx    
```
We would add a pre-launch command to check that the configured `plan` is a file.

Reply:
I changed it because you can't open the template version in jmeter gui but it's not a big deal as it's one line of sed to switch between.